src/osgPlugins/trans/ReaderWriterTRANS.cpp

```diff
@@ -81,7 +81,7 @@ static bool getFilenameAndParams(const std::string& input, std::string& filename
  * This pseudo-loader make it simple to change the origin of a saved model
  * by specifying a correcting translation as part of the filename.
  *
- * Usage: <modelfile.ext>.<tx>,<ty>,<tz>.globe
```

Comment:
Again, what's going on here?

Reply:
This is correcting what looks like a copy and paste error in the comment describing the usage of the TRANS plugin.